src/mame/sinclair/pentevo.cpp

```diff
@@ -119,6 +124,10 @@ class pentevo_state : public atm_state
 	required_device<spi_sdcard_sdhc_device> m_sdcard;
 	required_device<at_keyboard_device> m_keyboard;
 	required_ioport_array<3> m_io_mouse;
+	required_device_array<ym2149_device, 2> m_ay;
+
+	int m_ay_selected;
```

Comment:
This should be an explicitly sized integer type as it needs to be registered for save states.